libmachine/provision/configure_swarm.go

```diff
@@ -117,17 +118,23 @@ func configureSwarm(p Provisioner, swarmOptions swarm.Options, authOptions auth.
 			Name:              "always",
 			MaximumRetryCount: 0,
 		},
+		Binds: []string{hostBind},
```

Comment:
In my case, the nodes need to access the certs for authenticating with the consul. I either need to bind them like this or be able to specify a volume option (-v /mycerts:...) and this seemed like the easiest way (I also understand that this isn't the best approach for a long term solution).